src/testdrive/src/action.rs

```diff
@@ -338,6 +338,32 @@ impl State {
             }
         }
 
+        // Grant initial privileges.
```

Comment:
This part should be run in any testdrive usage, right? I might have to add testdrive executable to coverage too or ignore src/testdrive subdirectory.